### PR TITLE
added python notes and examples to kinematics intro

### PR DESCRIPTION
**Pull request by jasondaming**

### CODE REVIEW

File: source/docs/software/kinematics-and-odometry/intro-and-chassis-speeds.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 What is odometry?
 -----------------
 Odometry involves using sensors on the robot to create an estimate of the position of the robot on the field. In FRC, these sensors are typically several encoders (the exact number depends on the drive type) and a gyroscope to measure robot angle. The odometry classes utilize the kinematics classes along with periodic user inputs about speeds (and angles in the case of swerve) to create an estimate of the robot's location on the field.
 
-The Chassis Speeds Class
+The ChassisSpeeds Class
 ------------------------
```
Comment (sciencewhiz) [inline source/docs/software/kinematics-and-odometry/intro-and-chassis-speeds.rst:13]: need to update length

Comment (jasondaming) [reply source/docs/software/kinematics-and-odometry/intro-and-chassis-speeds.rst]: done thanks

File: source/conf.py [hunk 2]
```diff
@@ -323,3 +324,19 @@ def new_send(self, data):
 
 
 http.client.HTTPConnection.send = new_send
+
+intersphinx_mapping = {
+    "robotpy": ("https://robotpy.readthedocs.io/en/stable/", None),
```
Comment (virtuald) [inline source/conf.py:329]: Should be able to remove both of these and only link to https://robotpy.readthedocs.io/projects/robotpy/en/latest/ , which is the combined mostrobotpy api site. It doesn't include commands, but that can be added later if needed.

**Comment by jasondaming** [comment]
Assuming this is good I think we should push this soon so that we can use the intersphinx stuff it has on other docs

File: source/docs/software/kinematics-and-odometry/intro-and-chassis-speeds.rst [hunk 3]
```diff
@@ -40,12 +40,22 @@ The constructor for the ``ChassisSpeeds`` object is very straightforward, accept
       frc::ChassisSpeeds speeds{3.0_mps, -2.0_mps,
         units::radians_per_second_t(std::numbers::pi)};
 
+   .. code-block:: python
+
+      import math
+      from wpimath.kinematics import ChassisSpeeds
+
+      // The robot is moving at 3 meters per second forward, 2 meters
```
Comment (virtuald) [inline source/docs/software/kinematics-and-odometry/intro-and-chassis-speeds.rst:48]: Python comments are `#`, not `//`

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**